media/src/main/java/io/wcm/handler/media/impl/MediaFormatResolver.java

```diff
@@ -165,25 +168,20 @@ private boolean addResponsiveImageMediaFormats(MediaArgs mediaArgs) {
 
   private boolean resolveForImageSizes(MediaArgs mediaArgs, Map<String, MediaFormatOption> additionalMediaFormats) {
     ImageSizes imageSizes = mediaArgs.getImageSizes();
-    if (imageSizes == null) {
+    if (imageSizes == null || imageSizes.getWidthOptions() == null) {
```

Comment:
getWidthOption is never null (marked with NonNull - check generates a warning)

Reply:
done